sklearn/tests/test_pipeline.py

```diff
@@ -8,6 +8,7 @@
 import itertools
 
 import pytest
+import warnings
```

Comment:
[CI linting error](https://dev.azure.com/scikit-learn/scikit-learn/_build/results?buildId=40802&view=logs&j=32e2e1bb-a28f-5b18-6cfc-3f01273f5609&t=8a54543f-0728-5134-6642-bedd98e03dd0):

```suggestion
```

Reply:
Thanks. Have fixed this error.